_posts/2025-05-04-cambios-formato-moneda.md

```
@@ -0,0 +1,21 @@
+tags: [Reportes, Cliente, Solicitudes, Matriz de riesgo]
+comments: false
+author: Catalina Pinilla
+---
+
+## Separación de deuda DICOM por monedas
+
+### Matriz de riesgo
+
+Desde ahora en la Matriz de riesgo puedes ver que si tu cliente trabaja con distintas monedas, Advance ahora te mostrará el resumen de los distintos totales por cada uno de los totales según las monedas que utiliza y luego al extender la tabla podrás ver el detalle ordenado por categoría, viendo por defecto el peso chileno primero, esto se puede organizar según tú preferencia.
+
+<video width="100%" controls>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: video

(MD033, no-inline-html)